R/govList.R

```diff
@@ -0,0 +1,54 @@
+#' @rdname layouts
+#' @param list vector of list
+#' @param style options: "none", "bullet", "number". defaults to "none".
+#' @export
```

Comment:
Thanks Jeni! Please can you add in an example to the govList R file like how the other functions are set up? Then running devtools::check() should update the documentation, should be good to go after that though!